code/datums/wires/airlock.dm

```diff
@@ -55,9 +55,12 @@ var/const/AIRLOCK_WIRE_LIGHT = 512
 /datum/wires/airlock/CanUse(mob/living/L)
 	var/obj/machinery/door/airlock/A = holder
 	if(iscarbon(L))
-		if(A.isElectrified())
-			if(A.shock(L, 100))
-				return 0
+		if(A.x - L.x <= 1 && A.x - L.x >= -1 && A.y - L.y <= 1 && A.y - L.y >= -1)
+			if(A.isElectrified())
+				if(A.shock(L, 100))
+					return 0
+		else
```

Comment:
This would make it so that it always return false if you're a carbon and it isn't electrified.

Reply:
Works as intended, as additional failsafe in case someone manages to somehow reproduce the issue regardless of adjacency check.
But, seeing as it does make UI behave somewhat unexpectedly and inconsistently with other UIs, I guess removing this is appropriate.